vector/src/main/java/im/vector/app/features/spaces/invite/SpaceInviteBottomSheetViewModel.kt

```diff
@@ -57,6 +61,38 @@ class SpaceInviteBottomSheetViewModel @AssistedInject constructor(
                         peopleYouKnow = Success(peopleYouKnow)
                 )
             }
+            refreshInviteSummaryIfNeeded(roomSummary)
+        }
+    }
+
+    private fun refreshInviteSummaryIfNeeded(roomSummary: RoomSummary) {
+        if (roomSummary.membership == Membership.INVITE) {
+            // we can try to query the room summary api to get more info?
+            viewModelScope.launch(Dispatchers.IO) {
+                when (val peekResult = tryOrNull { session.peekRoom(roomSummary.roomId) }) {
+                    is PeekResult.Success -> {
+                        setState {
+                            copy(
+                                    summary = Success(
+                                            roomSummary.copy(
+                                                    joinedMembersCount = peekResult.numJoinedMembers,
```

Comment:
Maybe have a small method to map PeekResult to RoomSummary also?

Reply:
Here we are copying an existing roomSummary with info from peekResult, so I think this is not applicable